libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/NavigationInstructionPlayer.java

```diff
@@ -16,16 +16,23 @@
   private AndroidSpeechPlayer androidSpeechPlayer;
   private VoiceInstructionMilestone voiceInstructionMilestone;
   private boolean isMuted;
+  private boolean languageSupportedByMapboxSpeech;
 
-  public NavigationInstructionPlayer(@NonNull Context context, String language, String accessToken) {
+  public NavigationInstructionPlayer(@NonNull Context context, String language,
+                                     boolean languageSupportedByMapboxSpeech, String accessToken) {
+    this.languageSupportedByMapboxSpeech = languageSupportedByMapboxSpeech;
     initAudioManager(context);
     initAudioFocusRequest();
     initInstructionPlayers(context, language, accessToken);
   }
 
   public void play(VoiceInstructionMilestone voiceInstructionMilestone) {
     this.voiceInstructionMilestone = voiceInstructionMilestone;
-    mapboxSpeechPlayer.play(voiceInstructionMilestone.getSsmlAnnouncement());
+    if (languageSupportedByMapboxSpeech) {
```

Comment:
Could we extract this `if else` block into a `private` well-named method? Even better, could we simplify the conditional (`if else` ) somehow? Ideally removing it would be 💯